doc/GUIDE.md

```diff
@@ -340,7 +340,7 @@ does so automatically.
 ### Listing Dependencies
 
 Let's have stack add a few more dependencies to our project. First, we'll include two new packages in the
-`build-depends` section for our library in our `helloworld.cabal`:
+`build-depends` section for our library in our `package.yaml``:
```

Comment:
Looks like an extra backtick snuck in after `package.yaml`, unless it's just the diff looking funny.

Reply:
Good :eyes:, amended.
Also actually checked [here](https://github.com/dukedave/stack/blob/use-hpack-in-guide/doc/GUIDE.md) and LGTM.